examples/CMakeLists.txt

```diff
@@ -3,21 +3,16 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 # cmake-format: on
 
-set(BEMAN_OPTIONAL26_LIBRARY "beman_execution26")
+set(BEMAN_EXECUTION26_LIBRARY beman_execution26)
```

Comment:
I removed quotes in several places where they seemed superficial. Please let me know if they served a real purpose somewhere that I overlooked.